src/renderer/i18n/en/app.json

```diff
@@ -425,6 +425,7 @@
     },
     "deviceStorage": {
       "freeSpace": "<0>{{space}}</0> free",
+      "noFreeSpace": "no storage left",
```

Comment:
@dasilvarosa is this ok?

Reply:
@juan-cortes Cannot say without a screenshot